src/ngraph/runtime/backend.cpp

```diff
@@ -29,6 +29,11 @@ runtime::Backend::~Backend()
 {
 }
 
+DL_HANDLE runtime::Backend::get_handle(const std::string& type)
```

Comment:
I'd prefer returning a defined interface that backends can implement rather than leaving the keys to the backdoor of the dll the under the mat like a handle does.